sponsorship-prospectus.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: default
+header: Sponsorship Prospectus
+title: Sponsorship Prospectus
+nav: sponsorship-prospectus
+bloglike: true
+---
+
+## Introduction
+
+PyCon India is the largest gathering of Pythonistas in India for the Python
+programming language. The 12th edition of PyCon India will be taking place
+online from 2nd October to 5th October 2020.
+
+With this unique scenario at hand, we plan to make this year's conference
+bigger, better, and more accessible to Pythonistas all across the world
+who can watch, participate and share their views right from the comfort
+of their homes.
+
+
+<br>
+
+## PyCon India 2020
+
+- 12th year of PyCon India
+- It will be a 4 day event
+  - Workshops: Oct 4th (Sun), 2020
+  - Conferences: Oct 2th (Fri) - Oct 3th (Sat), 2020
+  - Developer Sprints: Oct 4th (Sun) - Oct 5th (Mon), 2020
+- Estimated Participation: 1500+
+- Online Community Presence
+  - Local User groups : 10,000+ (via different local mailing lists)
+  - Facebook: 6500+ (likes and followers)
+  - Twitter: 9000+
+
+
+<br>
+
+## Why Should You Sponsor?
+
+### Put Your Business in the Spotlight
+
+Be a part of India’s Premiere conference on using and
+developing the Python Programming Language, increasing your
+brand exposure and elevating your business identity within the
+community. Stand out among the attendees by prominently
+displaying your products and marketing material.
+
+### Hiring and Business Opportunities
+
+Get matched with those who could become potential clients or
+employees. The conference has a history of attracting smart
+folks and businesses from various backgrounds. Get connected,
+make your pitch and find the best programmers or clients to
+join your ranks!
+
+### Give Back to the Community
+
+Establish your goodwill and show the community that you’re a
+reliable partner, willing to support the
+ecosystem. Sponsorship is a way of giving back to the
+community and thanking them for their support. Your
+sponsorship helps PyCon India remain affordable and accessible
+to the widest possible audience.
+
+### Low Cost Marketing
+
+Instead of spending a fortune on traditional advertisements,
+take a look at our reasonably priced sponsorship slabs and
+pick one - letting you be discovered and known to the cream of
+the community at maybe half (or even less!) of your marketing
+budget.
+
+### Flexibility
+
+Have something specific in mind? Please don’t hesitate to ask!
+We are more than happy to work with you and tailor the
+sponsorship benefits to suit your needs.
+
+
+### Audience profile
+
+PyCon India is attended by a diverse mix of people across
+domains and expertise which includes and not limited to -
+Developers, technologists, programmers, educators, bloggers,
+authors, web developers, CTOs, managers, entrepreneurs,
+scientists, engineers and domain experts.
+
+<br>
+
+## Sponsorship Packages
+
+<table class="table table-striped">
```

Comment:
The table columns are rendering too close in the netlify build. It looks kinda odd. 
Also, it looks like plaintext. Not table rows. There are no borders in the table.

Reply:
I have no idea how to add border in the table using jekyll. Do we want add table, th, td {
  border: 1px solid black;
  border-collapse: collapse;
} in assets/css/style.css or something else?